examples/plot_gait.py

```diff
@@ -0,0 +1,241 @@
+# overlaid on the same plot
+
+fig, ax = plt.subplots()
+ax.plot(trial['data']['LAV'][5000:5800],
+        label='left foot vertical acceleration')
+ax.plot(trial['data']['RAV'][5000:5800],
+        label='right foot vertical acceleration')
+ax.set_xlabel('time (x10ms)')
+ax.set_ylabel('acceleration ($m.s^{-2}$)')
+ax.legend()
+
+###############################################################################
+# We can see the alternating left and right foot movements.
+#
+# In the rest of this example, we will only use the right foot
+# vertical acceleration.
+
+###############################################################################
+# Convolutional Dictionary Learning
+# ---------------------------------
+#
+# Now, let's use DiCoDiLe to learn patterns from the data and reconstruct
+# the signal from a sparse representation.
+#
+# First, we initialize a dictionary from parts of the signal:
+
+X = trial['data']['RAV']
+X = X.reshape(1, *X.shape)
+
+print(X.shape)
+
+D_init = init_dictionary(X, n_atoms=8, atom_support=(300,), random_state=60)
+
+###############################################################################
+# Note the use of ``reshape`` to shape the signal as per ``dicodile``
+# requirements: the shape of the signal should be
+# ``(n_channels, *sig_support)``.
+# Here, we have a single-channel time series.
+
+###############################################################################
+# Then, we run DiCoDiLe!
+
+D_hat, z_hat, pobj, times = dicodile(X, D_init, n_iter=3,
+                                     n_workers=4,
+                                     dicod_kwargs={"max_iter": 10000},
+                                     verbose=6,
+                                     window=True)
+
+
+print("[DiCoDiLe] final cost : {}".format(pobj))
+
+###############################################################################
+# We'll now display the initial and final dictionary side by side
+
+# normalize dictionaries
+normalized_D_init = D_init / D_init.max()
+normalized_D_hat = D_hat / D_hat.max()
+
+fig = display_dictionaries(normalized_D_init, normalized_D_hat)
+
+###############################################################################
+# We can order the dictionary patches by decreasing sum of the activations'
+# absolute values in the activations ``z_hat``, which, intuitively, gives
+# a measure of how they contribute to the reconstruction.
+
+sum_abs_val = np.sum(np.abs(z_hat), axis=-1)
+sum_abs_val
+
+
+""
+# we negate sum_abs_val to sort in decreasing order
+patch_indices = np.argsort(-sum_abs_val)
+
+fig_reordered = display_dictionaries(normalized_D_init[patch_indices],
+                                     normalized_D_hat[patch_indices])
+
+###############################################################################
+# Signal reconstruction
+# ^^^^^^^^^^^^^^^^^^^^^
+#
+# Now, let's reconstruct the original signal
+
+X_hat = reconstruct(z_hat, D_hat)
+
+""
```

Comment:
I would remove this line

Reply:
Isn't the reconstruction fairly important?

Reply:
I mean remove the line with the `""`

Reply:
OK, I think (guess) jupytext uses that as a cell split marker, but I can remove it and add some prose between the reconstruction and the plot?